runtime/convertValues.go

```diff
@@ -112,6 +112,8 @@ func exportValueWithInterpreter(
 		return cadence.UFix64(v), nil
 	case *interpreter.CompositeValue:
 		return exportCompositeValue(v, inter, seenReferences)
+	case *interpreter.SimpleCompositeValue:
```

Comment:
`SimpleCompositeValue` is a new value that is used for representing composite values that are built-in and not stored. Some of those built-in values are exportable